README.md

```diff
@@ -32,13 +32,32 @@ React.render(
 );
 ```
 
+Gist from URL:
+
+***NOTE: File cannot be parse from the URL. Supply as separate property.***
```

Comment:
Any reason to this?
I'd say it would be best to get the filename from the url as well.

From a use case perspective, would a user know the url without the file, or everything at once? Like the following url: `https://gist.github.com/tleunen/c10bcdb1f225f08316d49ad57bb13d3d#file-create-reducer-js`

Reply:
Hi @tleunen,

I think it's because you can’t extract the filename from the URL as he said.

As example, by parsing the filename from the url `https://gist.github.com/ringods/5995ea726914f280afb3#file-chef-dockerfile`, we would get `file-chef-dockerfile` or maybe `chef-dockerfile` instead of the real filename which is `Chef-Dockerfile`.

So, if someone needs to get the real filename on the `iframe#id`, I think this developer would have to use the `Gist#file` property.

What do you think?

Reply:
Hi @tleunen, any news? :/